style.css

```diff
@@ -1,7 +1,802 @@
+    height: 40px;
+    background-color: var(--primary-color);
+    font-family: var(--tertiary-font);
+    font-weight: 600;
+  }
+
+  .header a {
+    text-decoration: none;
+    color: #fff;
+    font-size: 14px;
+  }
+
+  .header a:hover {
+    color: var(--secondary-color);
+  }
+
+  .logo img {
+    width: 70px;
+  }
+
+  .social img {
+    width: 25px;
+  }
+
+  .main-program img {
+    width: 30px;
+    height: auto;
+  }
+
+  .grid-item img {
+    height: 150px;
+    background-image: url("/assets/images/patern.png");
+    background-size: 70px;
+    background-repeat: no-repeat;
+    padding: 15px 0 0 15px;
+  }
+
+  .partner img {
+    padding-top: 20px;
+    padding-bottom: 10px;
+    width: 80px;
+  }
+
+  .social img:hover {
+    background-color: var(--secondary-color);
+  }
+
+  /*****  navbar  ****/
+  #navbar {
+    position: static;
+    display: flex;
+    transform: translateX(0);
+    background-color: #fff;
+    height: 70px;
+    width: 100%;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 0 10%;
+    align-items: center;
+    box-shadow: none;
+  }
+
+  .logo {
+    display: block;
+    width: 70px;
+  }
+
+  #navbar ul {
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    padding: 20px 0;
+    gap: 30px;
+  }
+
+  #navbar ul li {
+    border: none;
+    padding: 0;
+    font-size: 14px;
+    font-weight: 600;
+  }
+
+  .focused {
+    padding: 10px !important;
+    margin: 0;
+    box-shadow: none;
+    border: solid 2px var(--secondary-color) !important;
+  }
+
+  .focused:hover {
+    border: none;
+    color: var(--tertiary-color);
+  }
+
+  .humberger {
+    display: none;
+  }
+
+  .humberger div {
+    display: none;
+  }
+
+  /************  Intro Section  ************/
+
+  .intro {
+    height: auto;
+    padding: 30px 120px;
+  }
+
+  .intro h2 {
+    padding-top: 50px;
+    font-size: 34px;
+  }
+
+  .intro h1 {
+    width: 80%;
+    font-size: 56px;
+    line-height: 3.5rem;
+  }
+
+  .intro .bio {
+    width: 65%;
+    font-size: 16px;
+    line-height: 1.5rem;
+  }
+
+  .intro .time {
+    font-weight: 800;
+    font-size: 28px;
+  }
+
+  .intro .address {
+    padding-top: 20px;
+    font-family: var(--secondary-font);
+    font-size: 18px;
+    padding-bottom: 70px;
+  }
+
+  /******  Main program Section  ******/
+  #main-program {
+    padding: 20px 50px 50px;
+  }
+
+  #main-program .title {
+    font-size: 20px;
+    font-weight: 700;
+    padding-top: 20px;
+    padding-bottom: 10px;
+  }
+
+  .main-program {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+    column-gap: 3px;
+  }
+
+  .main-program div {
+    display: grid;
+    align-items: start;
+    justify-content: center;
+    justify-items: center;
+    grid-template-rows: 20px 10px auto;
+  }
+
+  .main-program div:hover {
+    border: solid 3px var(--tertiary-color);
+  }
+
+  .main-program h3 {
+    color: var(--secondary-color);
+
+    /* flex: 1.2 1 0; */
```

Comment:
- Kindly remove the unused commented line of code to make your codebase look cleaner.

Reply:
Thank you so much, Ill note them all and consider changing them